testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/social/MicrosoftLoginPage.java

```diff
@@ -35,6 +35,12 @@ public class MicrosoftLoginPage extends AbstractSocialLoginPage {
     @FindBy(id = "idSIButton9")
     private WebElement submitButton;
 
+    @FindBy(xpath = "//input[contains(@class,'button_primary')]")
+    private WebElement staySignedIn;
```

Comment:
I am aware that same button could be used as they have same ID but I think it shouldn't be done that way as it might be confusing for somebody who haven't debug test yet, leaving them with impression that the same button was clicked twice which isn't the case. WDYT ?

Reply:
Considering this it's probably better to stick to approach previously used which you suggested, fixed